ui/src/app/app-routing.tsx

```diff
@@ -127,7 +128,10 @@ export const AppRoutingComponent: React.FunctionComponent<RoutingProps> = ({onSi
         path='/'
           component={() => <HomepagePage routeData={{title: 'Homepage'}}/>}
       />
-      <AppRoute path='/access-renewal' component={() => <AccessRenewalPage routeData={{title: 'Access Renewal'}}/>}/>
+      <AppRoute path='/access-renewal' component={() => serverConfigStore.get().config.enableAccessRenewal
```

Comment:
Tested on Test-Local: redirects me to profile
Tested on Local-Local: can access this page
👍